src/target.cpp

```diff
@@ -1450,6 +1450,7 @@ const char *arch_stack_pointer_register_name(ZigLLVM_ArchType arch) {
         case ZigLLVM_aarch64_32:
         case ZigLLVM_riscv32:
         case ZigLLVM_riscv64:
+        case ZigLLVM_mipsel:
```

Comment:
Looks like it's safe to put at least `ZigLLVM_mips` here as well, yeah?

Reply:
Yup, I didn't add it because I've only tested the LE variant.